spirv_hlsl.hpp

```diff
@@ -54,8 +54,6 @@ enum HLSLBindingFlagBits
 	HLSL_BINDING_AUTO_NONE_BIT = 0,
 
 	// Push constant (root constant) resources will be declared as CBVs (b-space) without a register() declaration.
-	// A register will be automatically assigned by the D3D compiler, but must therefore be reflected in D3D-land.
```

Comment:
This comment is still correct. If AUTO_PUSH_CONSTANT_BIT is used that is the intent.